From 9d36210beca09cc7a3841d09e5246d4b0000c52d Mon Sep 17 00:00:00 2001
From: Tanu Kaskinen <tanuk@iki.fi>
Date: Tue, 17 Mar 2020 09:26:57 +0200
Subject: [PATCH] ucm: Don't log errors during normal operation

It's completely normal to not have explicit channel configuration for
stereo devices. In fact, the ALSA developers actively avoid configuring
the channels for stereo devices.

I also dropped the word "duplex" from the messages, because "stereo
duplex" implies bidirectionality, but most devices use one direction
only.
---
 src/modules/alsa/alsa-ucm.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/modules/alsa/alsa-ucm.c b/src/modules/alsa/alsa-ucm.c
index 8df24e7b51d7..37cbafae51ee 100644
--- a/src/modules/alsa/alsa-ucm.c
+++ b/src/modules/alsa/alsa-ucm.c
@@ -379,8 +379,8 @@ static int ucm_get_device_property(
 
     if (pa_proplist_gets(device->proplist, PA_ALSA_PROP_UCM_SINK) &&
         device->playback_channels == 0) {
-        pa_log("UCM file does not specify 'PlaybackChannels' "
-                    "for device %s, assuming stereo duplex.", device_name);
+        pa_log_info("UCM file does not specify 'PlaybackChannels' "
+                    "for device %s, assuming stereo.", device_name);
         device->playback_channels = 2;
     }
 
@@ -400,8 +400,8 @@ static int ucm_get_device_property(
 
     if (pa_proplist_gets(device->proplist, PA_ALSA_PROP_UCM_SOURCE) &&
         device->capture_channels == 0) {
-        pa_log("UCM file does not specify 'CaptureChannels' "
-                    "for device %s, assuming stereo duplex.", device_name);
+        pa_log_info("UCM file does not specify 'CaptureChannels' "
+                    "for device %s, assuming stereo.", device_name);
         device->capture_channels = 2;
     }
 
-- 
2.16.4

